templates/components/messages_after_redirect_alerts.html.twig

```diff
@@ -39,21 +41,11 @@
       {% set class = '' %}
       {% set title = '' %}
       {% if type == constant('ERROR') %}
-         {% set class = 'alert-danger' %}
-         {% set title = _n('Error', 'Errors', 1) %}
+          {{ alerts.alert_danger(_n('Error', 'Errors', 1), [mesage|raw], true) }}
```

Comment:
I think `|raw`  has no effect when passing arguments to a macro.

```suggestion
          {{ alerts.alert_danger(_n('Error', 'Errors', 1), [mesage], true) }}
```

The same remark applies to other macro calls in the current file.